core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -850,7 +856,14 @@ public function copy($source=null, $destination=null)
 			throw new InternalServerErrorException('Attempt to copy the folder "' . $source . '" to "' . $strFolder . '" (circular reference).');
 		}
 
-		$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_COPY, new DataContainerSubject($this->strTable, $source, array('destination' => $destination)));
```

Comment:
On a side note, why does a non-existing folder give an AccessDenied instead of a 404 not found? @contao/developers 😆

Reply:
see https://github.com/contao/contao/pull/4826